metadata_tools/hosts/GO_0xxx/GO_0xxx_index_cloud.py

```
@@ -0,0 +1,82 @@
+#
+#   cloud_tasks stop --config gcp_index_config.yml
+#   cloud_tasks purge_queue --config gcp_index_config.yml
+#   cloud_tasks delete_queue --config gcp_index_config.yml
+#
+#   cloud_tasks show_queue --project-id rms-metadata --job-id metadata-index-job --provider gcp --detail
+#   cloud_tasks status --project-id rms-metadata --job-id metadata-index-job --provider gcp
+#   cloud_tasks list_running_instances --project-id rms-metadata --job-id metadata-index-job --provider gcp
+#
+#########################################################################################
+import asyncio
+import os, sys
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)